src/App.tsx

```diff
@@ -15,71 +16,94 @@ export const goods = [
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  // eslint-disable-next-line @typescript-eslint/no-shadow
+  AddButton = (goods: string) => {
+    this.setState({ selectedGood: goods });
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  render() {
+    const { selectedGood } = this.state;
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+    return (
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+      <main className="section container">
+        {
+          selectedGood === '' ? (
+            <h1 className="title">
+              No goods selected
+            </h1>
+          ) : (
+            <h1 className="title is-flex is-align-items-center">
+              {selectedGood}
+              {' '}
+              is selected
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={(this.RemoveButton)}
+              />
+            </h1>
+          )
+        }
 
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Garlic
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </main>
-);
+        <table className="table">
+          <tbody>
+            {goods.map((good) => {
+              return (
+                <tr
+                  ata-cy="Good"
+                  className={classNames(
+                    '',
+                    { 'has-background-success-light': good === selectedGood },
+                  )}
+                >
+                  {!selectedGood || good !== selectedGood ? (
+                    <td>
+                      <button
+                        data-cy="AddButton"
+                        type="button"
+                        className="button"
+                        onClick={() => this.AddButton(good)}
+                      >
+                        +
+                      </button>
+                    </td>
+                  ) : (
+                    <td>
+                      <button
+                        data-cy="RemoveButton"
+                        type="button"
+                        className="button is-info"
+                        onClick={this.RemoveButton}
```

Comment:
Replace 'onClick={this.RemoveButton}' with 'onClick={this.handleRemoveButtonClick}' to use the renamed function.